src/firebolt/common/util.py

```diff
@@ -37,15 +51,52 @@ def fix_url_schema(url: str) -> str:
     return url if url.startswith("http") else f"https://{url}"
 
 
-def async_to_sync(f: Callable) -> Callable:
+class AsyncJobThread:
+    def __init__(self) -> None:
+        self.loop: Optional[AbstractEventLoop] = None
+        self.result: Optional[Any] = None
+        self.exception: Optional[BaseException] = None
+        super().__init__()
```

Comment:
What's `super()` in this context, since we're not parenting off anything? Object?

Reply:
Yeah, that's redundant